lib/customer.rb

```diff
@@ -0,0 +1,30 @@
+require "csv"
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
```

Comment:
`email` and `addresss` don't need to be `attr_accessor`, creating readers is all that's needed.  Giving users more access to your instance variables than needed is a bad practice.